hello.js

```diff
@@ -1,7 +1,7 @@
-function hello() {
+function hello(word) {
```

Comment:
I see you added "word" as a parameter to this function.  We'll learn more about functions later (Chapter 10), but for now, just know that not all functions have parameters - like this one.   It already contains all the code it needs to run inside of the function's curly braces.   Your code will still run with the parameter left in, because extra code doesn't always break a program.  But to make code readable for other programmers, it's best to remove extra code wherever possible.